testing/src/cypress/e2e/02-institution/02-dashboard-institution-page.cy.ts

```diff
@@ -1,53 +1,57 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
+import ManageInstitutionObject from "../../page-objects/Institution-objects/ManageInstitutionObject";
 
-describe("Dashboard Page", () => {
-  const dashboardInstitutionObject = new DashboardInstitutionObject();
-  const institutionCustomCommand = new InstitutionCustomCommand();
+const dashboardObject = new DashboardInstitutionObject();
+const iCC = new InstitutionCustomCommand();
+const manageInstituionObject = new ManageInstitutionObject();
 
-  const url = Cypress.env("institutionURL");
+const LOGIN_URL = Cypress.env("TEST").BASE_URL + "/institution/login";
+const UNAME = Cypress.env("TEST").UNAME_1;
+const PASS = Cypress.env("TEST").PASS_1;
 
+describe("Dashboard Page", () => {
   beforeEach(() => {
-    cy.visit(url);
-    institutionCustomCommand.loginInstitution();
+    cy.visit(LOGIN_URL);
+    iCC.loginWithCredentials(UNAME, PASS);
   });
 
-  it("Verify that user redirect to dashboard page", () => {
+  it("Verify that user is redirected to dashboard page", () => {
     cy.url().should("include", "/dashboard");
+    dashboardObject.dashboardWelcomeMessage().should("be.visible");
+    dashboardObject
```

Comment:
Is multiple assertions in same test a common practice ? Asking for my understanding.